libs/blocks/media/media.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/*
+ * Media - v5.1
+ */
+
+import { decorateBlockBg, decorateBlockDaa, decorateText, getBlockSize } from '../../utils/utils.js';
+
+export default function init(el, daa = true) {
```

Comment:
We would never pass in daa via the block loader. There's never a second param, so this can be removed.